boards/at/at_start_f405/at_start_f405_defconfig

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable MPU
+CONFIG_ARM_MPU=n
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=n
```

Comment:
?? comment does not match what is happening, if the soc does not support these then it should be set in `Kconfig.defconfig` of the soc, and presumably these already default to `n` unless they are set to `y`